Gems/MotionMatching/Code/Source/PoseDataJointVelocities.cpp

```diff
@@ -129,78 +129,82 @@ namespace EMotionFX::MotionMatching
                 const AZ::Vector3 jointPosition = relativeToWorldTM.TransformPoint(jointModelTM.m_position);
 
                 const AZ::Vector3& velocity = m_velocities[i];
+                const AZ::Vector3 velocityWorldSpace = relativeToWorldTM.TransformVector(velocity);
 
-                const float scale = 0.15f;
-                const AZ::Vector3 velocityWorldSpace = relativeToWorldTM.TransformVector(velocity * scale);
-
-                DebugDrawVelocity(debugDisplay, jointPosition, velocityWorldSpace, color);
+                const float scale = 0.1f;
+                DebugDrawVelocity(debugDisplay, jointPosition, velocityWorldSpace * scale, color);
             }
         }
     }
 
-    void PoseDataJointVelocities::CalculateVelocity(MotionInstance* motionInstance, size_t relativeToJointIndex)
+    void PoseDataJointVelocities::CalculateVelocity(const ActorInstance* actorInstance, Motion* motion, float requestedSampleTime, size_t relativeToJointIndex)
     {
+        MotionDataSampleSettings sampleSettings;
```

Comment:
suggestion: Is it worth zero initializing this to make sure the fields are zeroed? `MotionDataSampleSettings sampleSettings{};`